clang/include/clang/Basic/AttrDocs.td

```diff
@@ -3702,20 +3702,32 @@ user-declared functions. For example:
 
 .. code-block:: c++
 
+    #include <map>
+    #include <string>
+
+    using namespace std::literals;
+
     // Returns m[key] if key is present, or default_value if not.
     template<typename T, typename U>
     const U &get_or_default(const std::map<T, U> &m [[clang::lifetimebound]],
```

Comment:
We don't need the `[[clang::lifetimebound]]` on `m` for this example, do we?

Reply:
I didn't change this one.
However, it makes sense to me since the function returns either a reference to a value in the map or the default value so both could be referred to from the return value.